releases.json

```diff
@@ -3436,6 +3436,14 @@
       "1.12-1"
     ]
   },
+  "taocpp-json": {
+    "dependency_names": [
+      "taocpp_json"
```

Comment:
The upstream dependency name is taocpp-json, with a dash not an underscore!

Reply:
my bad, I mimic how`abseil-cpp` was defined to make taocpp-json's files. So, I thought that dependency were the symbol defined in the *provide* section of wrap file...

There are many other entries with dependencies in the usual meaning too.

So, I looked in`releases.json` and found that many entries have no *dependency_names* node, (such *catch* or *mocklibc*). Does it imply that the wrap name is implicitly used as a dependency?

Can you help me clarifying this point?

Reply:
> my bad, I mimic how`abseil-cpp` was defined to make taocpp-json's files. So, I thought that dependency were the symbol defined in the _provide_ section of wrap file...

They are, indeed, the symbol defined in the provide section of the wrap file.'

For example, abseil-cpp provides the dependency symbol:

/usr/lib64/pkgconfig/absl_base.pc

and hence uses "absl_base" in its provide section.

Similarly, taocpp-json provides taocpp-json-config.cmake, and hence should list "taocpp-json" in the wrap releases, to annotate the fact that the provide section has been fixed to provide "taocpp-json".

> and found that many entries have no _dependency_names_ node, (such _catch_ or _mocklibc_).

They are very old and buggy.

Reply:
thanks you for the clarification